cfme/tests/cloud/test_providers.py

```diff
@@ -198,6 +198,18 @@ def instance_with_ssh_addition_template(appliance, provider):
     instance.delete()
 
 
+@pytest.fixture
+def stack_without_parameters(provider):
+    stack = provider.mgmt.create_stack(name=fauxfactory.gen_alpha(10),
+        template_url="https://s3-us-west-2.amazonaws.com/cloudformation-templates-us-west-2/Managed"
+                     "_EC2_Batch_Environment.template",
```

Comment:
Q: Can we pull this template from yamls?

Reply:
Well, sure. There is no problem, but this template url won't change. I am manually testing this for three years and url is still the same.

Reply:
While that is true, I think it is good to separate data from code.